.github/workflows/apidocstest.yml

```
@@ -0,0 +1,90 @@
+name: api docs
+on:
+  pull_request:
+    branches:
+      - master
+jobs:
+  test:
+    runs-on: ubuntu-16.04
+    env:
+      PIP_DISABLE_PIP_VERSION_CHECK: on
+
+      SENTRY_LIGHT_BUILD: 1
+      SENTRY_SKIP_BACKEND_VALIDATION: 1
+      MIGRATIONS_TEST_MIGRATE: 0
+
+      PYTEST_SENTRY_DSN: https://6fd5cfea2d4d46b182ad214ac7810508@sentry.io/2423079
+      PYTEST_ADDOPTS: "--reruns 5"
+
+      # The hostname used to communicate with the PostgreSQL from sentry
+      DATABASE_URL: postgresql://postgres:postgres@localhost/sentry
+
+    steps:
+      - name: Install System Dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y --no-install-recommends \
+            libxmlsec1-dev \
+            libmaxminddb-dev
+      - uses: actions/checkout@v2
```

```suggestion

      - uses: actions/checkout@v2
```

Could you add spacing like this for readability? There are other places in this file too.